exercises/darts/canonical-data.json

```diff
@@ -1,101 +1,99 @@
 {
   "exercise": "darts",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
This is a breaking change. Should be 2.0.0

Reply:
I don't think there is anything for me to do, as you've increased the version number right (and I'll remember for future that while functionaly equivalent - its structurally different)

Reply:
I've not increased it?

It really should be 2.0.0.

Reply:
Ah yes - momentary mental freeze, will resubmit